weaviate/templates/weaviatePersistentVolumeClaim.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: weaviate-persistence-data
+  labels:
+    app: weaviate
+spec:
+  accessModes:
+    # Only using one replication so far.
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 32Gi
```

Comment:
That seems like a rather small default for storage. Also, I think configuring the storage size through `values.yaml` is a must.